src/comm.jl

```diff
@@ -95,3 +95,10 @@ function Intercomm_merge(intercomm::Comm, flag::Bool)
     return newcomm
 end
 
+function Universe_size(comm::Comm)
+    rank = Ref{Cint}()
```

Comment:
it doesn't store the MPI rank, so we should call it something else.
```suggestion
    flag = Ref{Cint}()
```